pyannote/audio/pipelines/clustering.py

```diff
@@ -200,6 +232,19 @@ def __init__(self, metric: str = "cosine", expects_num_clusters: bool = False):
             ["Affinity", "Unnormalized", "RandomWalk", "GraphCut"]
         )
         self.eigengap = Categorical(["Ratio", "NormalizedDiff"])
+        self.spectral_min_embeddings = LogUniform(1, 100)
+
+        # Hyperparameters for refinement operations.
+        self.refinement_sequence = Categorical(
+            ["O", "G", "TS", "GTS", "TSD", "GTSD", "TSDN", "GTSDN",
+             "TSN", "GTSN", "CTSDN", "CGTSDN"])
+        self.gaussian_blur_sigma = Categorical([0, 1, 2, 3])
```

Comment:
Why did you choose to switch to integer values? Why not stick with Uniform(0, 3)? Does it make sense to make it even larger? 

What does 0 mean in this case? No blur?

Reply:
> Why did you choose to switch to integer values?

In my previous experiments we only tried 0/1/2/3. Never tried float values or even larger values. But sigma could actually be float. Changed back to Uniform.

> What does 0 mean in this case? No blur?

Right it would be no blur. It's just calling `scipy.ndimage.gaussian_filter`.